src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -161,15 +162,17 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
         // Check for this explicitly here.
         if (singleStrand1.nonEmpty) {
           val ss1Flag = singleStrand1.head.negativeStrand
-          val ss1MI   = singleStrand1.head.apply[String](ConsensusTags.MolecularId)
-          require(singleStrand1.forall(_.negativeStrand == ss1Flag),
-            s"Not all AB-R1s and BA-R2s were on the same strand for molecule with id: $ss1MI")
+          if (singleStrand1.exists(_.negativeStrand != ss1Flag)) {
+            rejectRecords(ab ++ ba, FilterCollision)
```

Comment:
Do you think it would be reasonable to add a `logger.debug()` here with the `MI` of the offending group?  I think that when this error is seen it might be nice to be able to see the group with collisions and confirm that it's not some other issue.